spec/GraphQLOverHTTP.md

```diff
@@ -183,7 +183,7 @@ using the JSON encoding for GraphQL requests:
 | ------------------ | --------------------------------------- |
 | `application/json` | Standard type for GraphQL JSON requests |
 
-And for GraphQL responses:
+And for _GraphQL responses_:
```

Comment:
The GraphQL spec always uses the singular for keywords (probably because of `spec-md` behaviors, but also to make matching/indexing easier), I suggest we stick with this pattern too:

```suggestion
And for a _GraphQL response_:
```